src/iris/api.py

```diff
@@ -1791,6 +1791,13 @@ def on_post(self, req, resp):
                                                    `current_step`, `active`, `application_id`, `bucket_id`)
                            VALUES (:plan_id, :created, :context, 0, :active, :application_id, :bucket_id)''',
                         data).lastrowid
+                    # adding additional context
+                    if self.enable_triage_context or plan_id in self.triage_allow_list:
+                        context['incident_id'] = incident_id
+                        context['created'] = data['created']
```

Comment:
if we're going to add these iris variables here we should add all the built-in iris context variables as well

iris.target (target user name)
iris.priority
iris.application (application name)
iris.plan (plan name)
iris.plan_id

iris.incident_id

iris.template (template name)